eth-bytecode-db/verifier-alliance-database/src/lib.rs

```diff
@@ -0,0 +1,132 @@
+/// Provides access to internal functions to access the database.
+/// They mostly do not provide transactions consistency, and require
+/// users to be care of transactions themselves.
+///
+/// Are not recommended to be used directly.
+/// Prefer methods exposed to the public instead.
+pub mod internal;
```

Comment:
it seems to not be used anywhere else, maybe make it private?

Reply:
It is used inside the integration tests. At the same time, I would like to leave the flexibility of using internal implementation details if user's code needs that. So I prefer to make it public at least for now